notebooks/HASP/WavelengthAdjustment/WavelengthAdjustment.ipynb

```diff
@@ -0,0 +1,2632 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #43.        # New path for file
</pre><p>use pathlib for file/directory manipulations for cohesiveness.</p>

<pre class="ql-syntax" spellcheck="false">for file_path in stis_data.glob("*sx1.fits*"):
    filename = fits.getval(file_path, "FILENAME")
    new_path = stis_unshifted / filename
    shutil.move(str(file_path, new_path)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/297/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>